src/arizona_tpl_render.erl

```diff
@@ -25,8 +25,8 @@ Renderer.
 -spec render_target(Target, Block, Changes, Assigns) -> Rendered
     when Target :: root | json:decode_value(),
          Block :: arizona_tpl_compile:block(),
-         Changes :: arizona_socket:changes(),
-         Assigns :: arizona_socket:assigns(),
+         Changes :: arizona_template_renderer:assigns(),
+         Assigns :: arizona_template_renderer:assigns(),
```

Comment:
This again, without semantic distinction, seems strange.

Reply:
Let's discuss about this [here](https://github.com/arizona-framework/arizona/pull/168/files/8da36f47fce3436c82d8da049b587d1717d61f45#r1677158934).